browser/directives/componentPanel.html

```diff
@@ -25,7 +25,8 @@
             </div>
           </div>
         </div>
-        <span id="{{item.keyName}}-name" class="product-name">{{item.productName}}</span><span id="{{item.keyName}}-version" class="product-version">{{item.version}}</span>
+        <span id="{{item.keyName}}-name" class="product-name">{{item.productName}}</span>
+        <span id="{{item.keyName}}-version" class="product-info">{{item.version}}&nbsp;|&nbsp;{{item.size}}</span>
```

Comment:
UI test errors related to change n this line. adding span tag per text element w/ or w/o id attribute 

``` html
<span id="{{item.keyName}}-version" class="product-info">{{item.version}}</span><span class="product-info">|</span><span id="{{item.keyName}}-download-size" class="product-info">{{item.size}}</span>
```

fixes the UI test errors.